socs/agents/pysmurf_controller/agent.py

```diff
@@ -336,6 +336,14 @@ def check_state(self, session, params=None):
                     num_active_channels=num_active_channels,
                 )
                 session.data.update(d)
+
+                data = {
+                    'timestamp': time.time(),
+                    'block_name': 'state_results',
+                    'data': {'open_g3stream': d['open_g3stream'],
+                             'num_active_channels': d['num_active_channels']}
+                }
+                self.agent.publish_to_feed('state_results', data)
```

Comment:
This looks good, but the `state_results` feed needs to be registered, up in the `__init__`:
https://github.com/simonsobs/socs/blob/d23277f23ef91edc7ac44dc098b782d8c8f5e50c/socs/agents/pysmurf_controller/agent.py#L132-L135